Cargo.toml

```diff
@@ -1,6 +1,8 @@
 [workspace]
 resolver = "2"
 members = [
+    "components/chainhook-sdk",
+    "components/chainhook-types",
```

Comment:
Ok. Basically this PR brings back `chainhook-sdk` and `chainhook-types`.

Reply:
exactly. The code has been copied and simplified (clippy, remove dead code, use axum instead of rocket, this is the most important commit of this PR https://github.com/hirosystems/clarinet/pull/2010/commits/e2de07f9434500b6be7d8ed2a3d37214fe916b30)